charts/kong/example-values/full-k4k8s-with-kong-enterprise.yaml

```diff
@@ -0,0 +1,206 @@
+# Kong for Kubernetes with Kong Enterprise with Enterprise features enabled and
+# exposed via TLS-enabled Ingresses. Before installing:
+# * Several settings (search for the string "you-must") require user-provided
+#   Secrets. These Secrets must be created before installation.
+# * Ingresses reference example "<service>.kong.example" hostnames. These must
+#   be changed to an actual hostname that resolve to your proxy.
+# * Ensure that your session configurations create cookies that are usable
+#   across your services. The admin session configuration must create cookies
+#   that are sent to both the admin API and Kong Manager, and any Dev Portal
+#   instances with authentication must create cookies that are sent to both
+#   the Portal and Portal API.
+
+image:
+  repository: kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition
+  tag: 1.5.0.2-alpine
+  pullSecrets:
+    # you-must add this: https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-docker-registry-access
+    - kong-enterprise-edition-docker
+
+env:
+  prefix: /kong_prefix/
+  database: postgres
+
+  password:
+    valueFrom:
+      secretKeyRef:
+        name: you-must-create-an-rbac-bootstrap-secret
+        key: you-must-replace-with-your-key
+
+admin:
+  enabled: true
+  annotations:
+    konghq.com/protocol: "https"
+
+  tls:
+    enabled: true
+    servicePort: 8444
+    containerPort: 8444
+    parameters:
+    - http2
+
+  ingress:
+    enabled: true
+    tls: you-must-create-an-admin-tls-secret
+    hostname: admin.kong.example
+    annotations:
+      kubernetes.io/ingress.class: "kong"
+    path: /
+
+proxy:
+  enabled: true
+  type: LoadBalancer
+  annotations: {}
+
+  http:
+    enabled: true
+    servicePort: 80
+    containerPort: 8000
+    parameters: []
+
+  tls:
+    enabled: true
+    servicePort: 443
+    containerPort: 8443
+    parameters:
+    - http2
+
+  stream: {}
+
+  ingress:
+    enabled: false
+    hosts: []
+    annotations: {}
+    path: /
+
+  externalIPs: []
+
+enterprise:
+  enabled: true
+  license_secret: you-must-create-a-kong-license-secret
+  vitals:
+    enabled: true
+  portal:
+    enabled: true
+  rbac:
+    enabled: true
+    admin_gui_auth: basic-auth
+    session_conf_secret: you-must-create-an-rbac-session-conf-secret
+    admin_gui_auth_conf_secret: you-must-create-an-admin-gui-auth-conf-secret
+  smtp:
+    enabled: false
+    portal_emails_from: none@example.com
+    portal_emails_reply_to: none@example.com
+    admin_emails_from: none@example.com
+    admin_emails_reply_to: none@example.com
+    smtp_admin_emails: none@example.com
+    smtp_host: smtp.example.com
+    smtp_port: 587
+    smtp_auth_type: nil
+    smtp_ssl: nil
+    smtp_starttls: true
```

Comment:
We should instead resort to SMTP mocking here for a simpler setup?

Reply:
It does; `enabled: false` ignores these and sets mocking on. The other settings are present here just to demonstrate that they exist and should be set if you wish to enable it.

Reply:
Got it. Thanks for the explanation.